prowler/lib/check/custom_checks_metadata.py

```diff
@@ -18,9 +18,71 @@
+                        "CheckTitle": {
+                            "type": "string",
+                        },
+                        "Description": {
+                            "type": "string",
+                        },
+                        "Risk": {
+                            "type": "string",
+                        },
+                        "RelatedUrl": {
+                            "type": "string",
+                        },
+                        "Remediation": {
+                            "type": "object",
+                            "properties": {
+                                "Code": {
+                                    "type": "object",
+                                    "properties": {
+                                        "CLI": {
+                                            "type": "string",
+                                        },
+                                        "NativeIaC": {
+                                            "type": "string",
+                                        },
+                                        "Other": {
+                                            "type": "string",
+                                        },
+                                        "Terraform": {
+                                            "type": "string",
+                                        },
+                                    },
+                                    "required": [
+                                        "CLI",
+                                        "NativeIaC",
+                                        "Other",
+                                        "Terraform",
+                                    ],
+                                },
+                                "Recommendation": {
+                                    "type": "object",
+                                    "properties": {
+                                        "Text": {
+                                            "type": "string",
+                                        },
+                                        "Url": {
+                                            "type": "string",
+                                        },
+                                    },
+                                    "required": ["Text", "Url"],
+                                    "additionalProperties": False,
+                                },
+                            },
+                            "required": ["Code", "Recommendation"],
+                            "additionalProperties": False,
+                        },
                     },
-                    "required": ["Severity"],
+                    "required": [
+                        "Severity",
+                        "CheckTitle",
+                        "Description",
+                        "Risk",
+                        "RelatedUrl",
+                        "Remediation",
+                    ],
```

Comment:
do we want to mark all as required? I think they should be optional.

Reply:
I don't think they should be required. If missing, then the values from the check's json should be used

Reply:
Yes, I totally agree with that.